test/DynamoCoreWpfTests/PresetViewModelTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Windows;
+
+using SystemTestServices;
+
+using Dynamo;
+using Dynamo.Controls;
+using Dynamo.Models;
+using Dynamo.Nodes;
+using Dynamo.Selection;
+using Dynamo.Services;
+using Dynamo.Tests;
+using Dynamo.ViewModels;
+using NUnit.Framework;
+
+namespace DynamoCoreWpfTests
+{
+    [TestFixture]
+    public class PresetViewModelTests : DynamoViewModelUnitTest
+    {
```

Comment:
Make sure you take a look at the `Setup` method for `DynamoVIewModelUnitTest`.  It will create a new `DynamoViewModel` (and load all of the libraries, etc) just to check whether this text is truncated in your unit test.  Doesn't that sound wrong?  

To me, that's not a really a good reason to have this subclass `DynamoViewModelUnitTest`.  Your test will likely run several orders of magnitude slower because you are subclassing this, but for no ostensible reason other than you didn't want to create a new file.  

This is really only a test of `PresetPrompt`.  Let's change the name of this class to `PresetPromptTests` and not subclass `DynamoViewModelUnitTest`.